samples/snippets/src/main/java/com/google/cloud/gameservices/samples/deployments/UpdateDeployment.java

```
@@ -0,0 +1,62 @@
+import com.google.cloud.gaming.v1alpha.GameServerDeployment;
+import com.google.cloud.gaming.v1alpha.GameServerDeploymentsServiceClient;
+import com.google.protobuf.Empty;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class UpdateDeployment {
+  public static void updateGameServerDeployment(String projectId, String deploymentId) {
+    // String projectId = "your-project-id";
+    // String deploymentId = "your-game-server-deployment-id";
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (GameServerDeploymentsServiceClient client = GameServerDeploymentsServiceClient.create()) {
+      String deploymentName =
+          String.format(
+              "projects/%s/locations/global/gameServerDeployments/%s", projectId, deploymentId);
+
+      GameServerDeployment deployment =
+          GameServerDeployment.newBuilder()
+              .setName(deploymentName)
+              .putLabels("key", "value")
+              .build();
+
+      FieldMask fieldMask = FieldMask.newBuilder().addPaths("labels").build();
+
+      OperationFuture<GameServerDeployment, Empty> call =
+          client.updateGameServerDeploymentAsync(deployment, fieldMask);
+
+      GameServerDeployment updated = call.get(1, TimeUnit.MINUTES);
+      System.out.println("Game Server Deployment updated: " + updated.getName());
+    } catch (IOException | InterruptedException | ExecutionException | TimeoutException e) {
+      System.err.println("Game Server Deployment update request unsuccessful.");
+      e.printStackTrace(System.err);
+    }
```

Why catch this rather than just let it propagate?
If you catch something, at the least, you should suggest what might of happened and what the user might do to mitigate the problem.